lib/plugins/aws/configCredentials/awsConfigCredentials.js

```diff
@@ -64,19 +65,13 @@ class AwsConfigCredentials {
     this.serverless.cli.log('Setting up AWS...');
     this.serverless.cli.log('Saving your AWS profile in "~/.aws/credentials"...');
 
-    // locate home directory on user's machine
-    const env = process.env;
-    const home = env.HOME ||
-      env.USERPROFILE ||
-      (env.HOMEPATH ? ((env.HOMEDRIVE || 'C:/') + env.HOMEPATH) : null);
-
-    if (!home) {
+    if (!os.homedir()) {
```

Comment:
![image](https://cloud.githubusercontent.com/assets/3817380/22695748/7b152458-ed54-11e6-9150-69ed0550a7fb.png)


Hm, added in v2.3.0 - means safe.

My reasoning was, that I noticed that unit tests are using `os.homedir()` but the code not. After `git blame` found out it was because after refactoring project structure file was just copy pasted, and tests were written after by a different person. So just minor inconsistency.